documentation/Query-Syntax.md

```diff
@@ -474,7 +485,7 @@ i.e `{user: "user"}`
 
 Tree structure:
 
-![Example 1 tree diagram](images/query-syntax-example-1.png)
+[[images/query-syntax-example-1.png|Example 1 tree diagram]]
```

Comment:
Not sure if that's how this should be showing? Also when I click it it's not finding....
![Screen Shot 2022-03-03 at 1 33 05 PM](https://user-images.githubusercontent.com/16461988/156473283-ab861dbf-4641-431e-b1da-ac108adfdc44.png)

Reply:
Yeah, that's super annoying, I'm not sure why these got auto-formatted like that. Have fixed the image links now, will be vigilant if it happens again. 

ed34da9c